src/CarbonAware.WebApi/test/integrationTests/CarbonAwareControllerTests.cs

```diff
@@ -215,9 +218,9 @@ public async Task EmissionsForecastsBatch_SupportedDataSources_ReturnsOk(DateTim
                 foreach (var forecast in forecasts!)
                 {
                     Assert.That(forecast.Location, Is.EqualTo(location));
-                    Assert.That(forecast.DataStartAt, Is.EqualTo(start));
-                    Assert.That(forecast.DataEndAt, Is.EqualTo(end));
-                    Assert.That(forecast.RequestedAt, Is.EqualTo(reqAt));
+                    Assert.That(forecast.DataStartAt, Is.EqualTo(startDate));
```

Comment:
@bderusha , I think what would make really valuable is to check for the meaning of the content. Today there isn't anything like that, since like you said, the data is not reflecting what is the backend. At least now, we can verify whether something is generated and the expected fields are populated. What the content means, it is hard to know now.